css/index.css

```diff
@@ -161,4 +161,34 @@ body {
     .content__box {
         padding: 25px 10px;
     }
+  }
+
+  @media only screen and (min-width: 800px) {
+      .photo--splash {
+          display: flex;
+      }
+      .container__splash {
+          flex-direction: row;
+      }
+
+      body {
+          max-width: 800px;
```

Comment:
a best practice is to only use the body selector to style global font styles.  a wrapper div with a class container can be used to wrap all of the content on the page.  when setting width it is best to use percentages as opposed to pixels for responsiveness.